dvtm.c

```diff
@@ -1318,13 +1318,19 @@ redraw(const char *args[]) {
 
 static void
 scrollback(const char *args[]) {
+	int div = 0;
 	if (!is_content_visible(sel))
 		return;
 
-	if (!args[0] || atoi(args[0]) < 0)
-		vt_scroll(sel->term, -sel->h/2);
+	if (args[0])
+		div = atoi(args[0]);
+	if (!div)
+		div = -2;
+
+	if (div > sel->h)
+		vt_scroll(sel->term,  abs(div)/div);
```

Comment:
This does not seem right. You compare div, which is essentially compile-time constant with buffer height.